lib/puppet/util/package/version/range.rb

```diff
@@ -15,11 +15,11 @@ class ValidationFailure < ArgumentError; end
     # * Regular Version strings
     #   * ex. `"1.0.0"`, `"1.2.3-pre"`
     # * Inequalities
-    #   * ex. `">1.0.0"`, `"<3.2.0"`, `">=4.0.0"`
+    #   * ex. `"> 1.0.0"`, `"< 3.2.0"`, `">= 4.0.0"`
```

Comment:
space is not mandatory, you could just add one example with space

Reply:
>     # Number and location of spaces does not affect the parsing.
>     # * ex. `"> 1.0.0", ">1.0.0 , <= 2.3.0", " >=  4.0.0" etc.`
>     #
I opted for adding above additional comment.